MavenArtifacts.md

```diff
@@ -0,0 +1,83 @@
+# WPILib Maven Artifacts
+
+WPILib publishes its built artifacts to our maven server for use by downstream projects. This document explains these locations, and the meanings of artifact names, classifiers and versions.
```

Comment:
Oxford comma after `classifiers`

Reply:
I've always hated the look of that, but sure.